mcs/build/config-default.make

```diff
@@ -9,7 +9,11 @@
 # Use UTF-8 as the default encoding for compilers
 CODEPAGE = 65001
 
+ifdef TEST_WITH_INTERPRETER
+RUNTIME_FLAGS = --interpreter
+else
 RUNTIME_FLAGS =
+endif
```

Comment:
I'd move it to the TEST_WITH_INTERPRETER if clause we have already right below.

Reply:
+1